### PR TITLE
allow specifying annotations on service account

### PR DESCRIPTION
**Pull request by michaelst**
#### Summary
This allows specifying annotations on the service account in the cosigned helm chart. This is needed to use workload identity with GKE.

#### Release Note
```release-note
Added `.Values.webhook.serviceAccount.annotations` to allow specifying service account annotations
```

### CODE REVIEW

**Review by cpanato — COMMENTED** [review]
please bump the chart version as well

File: charts/cosigned/templates/webhook/sa_webhook.yaml [hunk 1]
```diff
@@ -6,3 +6,7 @@ metadata:
     control-plane: {{ template "cosigned.fullname" . }}-webhook
   name: {{ template "cosigned.fullname" . }}-webhook
   namespace: {{ .Release.Namespace }}
+  {{- with .Values.serviceAccount.annotations }}
```
Comment (cpanato) [inline charts/cosigned/templates/webhook/sa_webhook.yaml:9]: this does not exist in the values.yaml file

the one I can see is `webhook.service. annotations`

Comment (michaelst) [reply charts/cosigned/templates/webhook/sa_webhook.yaml]: I bumped the chart version and add the block to the values.yaml file, thanks!

**Review by cpanato — APPROVED** [review]

**Review by sabre1041 — APPROVED** [review]
LGTM

**Pull request merged**